frontend-react/src/components/MessageTracker/MessageDetails.tsx

```diff
@@ -15,57 +19,132 @@ type MessageDetailsProps = {
     id: string | undefined;
 };
 
+const dataToAccordionItems = (props: {
+    id: string;
+    title: string;
+    data: WarningError[];
+}): AccordionItemProps[] => {
+    const results: AccordionItemProps[] = [];
+    if (props.data.length === 0) {
+        return [];
+    }
+    const items = (
+        <WarningsErrors title={""} data={props.data.map((md) => md.detail)} />
+    );
+    results.push({
+        id: props.id,
+        title: props.title,
+        expanded: false,
+        content: items,
+        headingLevel: "h3",
+        className: "",
+    });
+
+    return results;
+};
+
 export function MessageDetails() {
     const navigate = useNavigate();
     const { id } = useParams<MessageDetailsProps>();
     const { messageDetails } = useMessageDetails(id!!);
+    const warnings: WarningError[] = messageDetails
+        ? messageDetails.warnings
+        : [];
+    const errors: WarningError[] = messageDetails ? messageDetails.errors : [];
+
+    const fileUrl = messageDetails?.fileUrl ? messageDetails.fileUrl : "";
+    const { folderLocation, sendingOrg, fileName } = parseFileLocation(fileUrl);
 
     return (
         <>
             <div
                 className="grid-container margin-bottom-10"
                 data-testid="container"
             >
-                <div className="margin-bottom-5">
-                    <h1>Message Details</h1>
-                    <Button onClick={() => navigate(-1)} type="button">
+                <div>
+                    <h2>Message ID Search</h2>
+                    <Button
+                        onClick={() => navigate("/admin/message-tracker")}
```

Comment:
Not a huge deal, but since this is navigating somewhere, I think we should use a link instead...but our USLink component always applies link-specific styles 🤔 

Maybe we can just use react-router's Link and add the `usa-button` modifier class for now?  But we'd likely want to expand USLink to adopt button classes down the line 🤔 🤔 🤔

Reply:
I think we should leave it for now. I hesitated making this change to begin with since this will be going away. We will be converting this section to a breadcrumb.

Reply:
Sure, makes sense to me!